localization/xliff/enu/constants/localizedConstants.enu.xlf

```diff
@@ -251,9 +251,24 @@
         <trans-unit id="msgPromptFirewallRuleCreated">
             <source xml:lang="en">Firewall rule successfully created.</source>
         </trans-unit>
+        <trans-unit id="msgChooseQueryHistoryListing">
+            <source xml:lang="en">Choose Query History listing</source>
```

Comment:
the casing on `listing` is inconsistent with the spelling in the item below.  Is that intentional?

Reply:
..also, not sure that "listing" is a great noun here.  Where did it some from?  I would probably go with something like "script" or similar.